### DISCUSSION TITLE
`split-pane --wait`?

### DISCUSSION THREAD

**Discussion by mkuratczyk** [wezterm/wezterm#3903] [General]
Hi,

I'd like to run tests in a temporary pane. Something like this:
```
wezterm cli split-pane --bottom --percent 50 -- cargo test
```

However, `wezterm` closes this pane immediately after the tests complete, so I can't see the output/results. It feels like there should be `--wait` parameter to open a split pane and wait for input before closing.  Is there some other solution for this?

I also can't find a wait to work around this issue by running `read` after `cargo test` - I'd need to surround the command and all args with quotes, but then `wezterm` treats this as a single command and returns "command not found".
So this doesn't work
```
wezterm cli split-pane --bottom --percent 50 -- 'cargo test; read'
```
and neither does this (because now `read` is a command after `wezterm`, not after `cargo test`)
```
wezterm cli split-pane --bottom --percent 50 -- cargo test; read
```

I'd appreciate suggestions,

**Comment by nekowinston** [comment]
You were on the right track, this should work:
```sh
wezterm cli split-pane --bottom --percent 50 -- sh -c "cargo test; read"
```

**Reply by mkuratczyk** [reply]
Doh, of course, thanks! :)
